### ISSUE TITLE
Iframe embed Audio pan effects issue

### ISSUE THREAD

**Issue by dinesh290722** [jitsi/jitsi-meet#14910]
### What happened?

When I embed the jitis meet conference in my website I am also playing my personal audio tracks in the same page on the left of the jitis meet outside of thr metting but I performing left and right pan effects those are working without embeding jitis meet after embeding jitis meet it is not working 

### Platform

- [X] Chrome (or Chromium based)
- [X] Firefox
- [X] Safari
- [X] Other desktop browser
- [ ] Android browser
- [ ] iOS browser
- [ ] Electron app
- [ ] Android mobile app
- [ ] iOS mobile app
- [ ] Custom app using a mobile SDK

### Browser / app / sdk version

Browser

### Reproducibility

- [X] The problem is reproducible on meet.jit.si

**Comment by saghul** [comment]
Browsers usually just open the audio device once. As such, if you use stereo but then Jitsi Meet requests mono (the default) you'll end up downgraded to mono.

You can turn on stereo in Jitsi meet overriding the config with the following url parameters: https://github.com/jitsi/jitsi-meet/blob/bde28105f40d521c373789473bbe3a9da3d1fa1b/config.js#L205

**Comment by dinesh290722** [comment]
I already enable 
audioQuality: {
              stereo: true,
              opusMaxAverageBitrate: 510000,
              enableOpusDtx: true,
            }
But also it is not working I want to do anything to enable?

**Comment by saghul** [comment]
Are you using your own server? In general you don't need the last 2, but it's ok.

Can you please share the browser console logs?

**Comment by dinesh290722** [comment]
Yes I hosted in my own server and I could not get any issues in the logs

**Comment by dinesh290722** [comment]
Audio control streo is true will need to work u want to enable any other feature?

**Comment by saghul** [comment]
> Yes I hosted in my own server and I could not get any issues in the logs

I wanted to check if the audio contraints were correct.



> Audio control streo is true will need to work u want to enable any other feature?

Not sure I understand, can you rephrase?

**Comment by saghul** [comment]
Check the audio contraints in getUserMedia in `chrome://webrtc-internals`
